tests/detect_machine.sh

```diff
@@ -5,30 +5,6 @@ export ACCNR=${ACCNR:-nems}
 
 case $(hostname -f) in
 
-  g10a1.ncep.noaa.gov)     MACHINE_ID=wcoss ; export pex=1;; ### gyre 1
```

Comment:
please do not remove wcoss phase1/2 in this commit. There is an effort to run GFSv16 on phase1/2 until June. So I'd like to keep rt.sh working on phase 1/2 just in case we need to debug something.